assets/js/products-block.jsx

```diff
@@ -337,7 +337,6 @@ const ProductsBlockPreview = withAPIData( ( { attributes } ) => {
 
 	if ( 'specific' === display ) {
 		query.include = display_setting.join( ',' );
-		query.orderby = 'include';
```

Comment:
I think this needs `query.per_page = display_setting.length;` here (or something similar). 

Otherwise you get an API request like `/wc/v2/products?per_page=3&include=809,1089,1003,806,798` and only 3/5 products show (Jay mentioned having this issue, but this is the cause of it). Since the row slider is gone we need to automatically handle this for the user.